tests/mechanisms/IN_Ih_Halnes2011.mod

```diff
@@ -0,0 +1,62 @@
+:Ih current for thalamic interneurons
```

Comment:
Can you add a note in README.md file of this folder with details of where these mod files come from, date added and which files are added.

Reply:
I think Aurélien maybe already mentioned it -- Can you add licenses to each new file being added: hoc, mod, py. However, I am not sure about it if we need to do it for other files too?